test/ae_mdw/aexn_tokens_test.exs

```diff
@@ -0,0 +1,89 @@
+defmodule AeMdw.AexnTokensTest do
+  use ExUnit.Case
+
+  alias AeMdw.AexnTokens
+  alias AeMdw.Database
+  alias AeMdw.Db.Model
+  alias AeMdw.Error.Input, as: ErrInput
+  alias AeMdw.Validate
+
+  import AeMdwWeb.Helpers.AexnHelper, only: [enc_ct: 1]
+
+  require Model
+
+  describe "fetch_token/1" do
+    test "returns an AEX-9 contract meta info" do
+      contract_pk = <<9_123_456::256>>
+
+      aex9_meta_info = {name, symbol, decimals} = {"Token1", "TK1", 18}
+
+      txi = 1_123_456_789
+
+      m_aexn =
+        Model.aexn_contract(
+          index: {:aex9, contract_pk},
+          txi: txi,
+          meta_info: aex9_meta_info
+        )
+
+      Database.dirty_write(Model.AexnContract, m_aexn)
+
+      contract_id = enc_ct(contract_pk)
+
+      assert {:ok,
+              %{
+                name: ^name,
+                symbol: ^symbol,
+                decimals: ^decimals,
+                contract_txi: ^txi,
+                contract_id: ^contract_id
+              }} = AexnTokens.fetch_token({:aex9, contract_pk})
+    end
+
+    test "returns a AEX-141 contract meta info" do
+      contract_pk = <<141_123_456::256>>
+
+      aex141_meta_info =
+        {name, symbol, base_url, type} = {"AE Boots", "Boot", Faker.Internet.url(), :url}
```

Comment:
We're using the `Faker` library just to make up an internet URL? I would try to avoid bloating the app with libraries, even if it's just for testing purposes

Reply:
:D

Reply:
bloating with one library? (((: It is the first use but no problem, I will implement one module for that.